server/models/Address.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+exports.addressSchema = new mongoose.Schema({
+  pickup: {
+    type: String,
+    required: true,
+    default: null,
+  },
+  service: {
+    type: String,
+    required: true,
+    default: null,
+  },
```

Comment:
Not sure what this is for?

Reply:
It's the address the service will be rendered at. For example, taking care of puppers at the client's home vs the sitter's home vs maybe some kind of dog daycare facility.